pom.xml

```diff
@@ -511,11 +511,47 @@ under the License.
                 <pomExclude>pom.xml</pomExclude>
               </pomExcludes>
               <localRepositoryPath>${project.build.directory}/local-repo</localRepositoryPath>
+              <filterProperties>
+                <repository.proxy.url>${repository.proxy.url}</repository.proxy.url>
+              </filterProperties>
+              <addTestClassPath>true</addTestClassPath>
```

Comment:
Curiousity: What explicitly do you need?

Reply:
It seems to add no needed behavior...I can delete this config.

Reply:
Then drop it.